### PR TITLE
[OUDS] Docs: Change the classes that were missing

### PR DESCRIPTION
**Pull request by louismaximepiton**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

NA

#### Remaining tasks and questions

Questions:

Tasks:

#### Done list

The following was done in the PR:
- Changed many things that we forgot to change in the previous PRs since this one will certainly be the last one on the tokens.
- Add the font example thumbnails.

### Types of change

- New feature (non-breaking change which adds functionality)

### Live previews

- <https://deploy-preview-2806--boosted.netlify.app/>

### CODE REVIEW

File: site/static/docs/0.0/assets/img/examples/font.png
Comment (julien-deramond) [inline site/static/docs/0.0/assets/img/examples/font.png:1]: Be careful, the screenshots are done in light mode usually. And these are 2 different fixes (examples and size classes), so should be in 2 different commits for the Git history.

**Review by julien-deramond — APPROVED** [review]

**Comment by louismaximepiton** [comment]
Renamed the commits, merging as-is.
